vortex-layout/src/layouts/dict/writer/mod.rs

```diff
@@ -123,15 +122,15 @@ impl LayoutWriter for DelegatingDictLayoutWriter {
 
 #[derive(prost::Message)]
 pub struct DictLayoutMetadata {
-    #[prost(enumeration = "pb::PType", tag = "1")]
+    #[prost(enumeration = "PType", tag = "1")]
```

Comment:
this is a drive by that I need to separate out of this pr. We use for dtype serialisation and now rust version of ptype has necessary prost annotations so we can use it directly.